en/core-api/bank-transfer.md

```diff
@@ -354,27 +593,29 @@ Please add **bank_transfer** parameter during [API Request](/en/core-api/bank-tr
 ```
 <!-- tabs:end -->
 
-Parameter | Type | Required? | Description
+#### VA Number Specification
+
+Parameter | Type | Required | Description
 --- | --- | --- | ---
-BCA `va_number`| String | (optional) | Length should be within 1 to 11.
-BCA `sub_company_code` | String | (optional) | BCA sub company code directed for this transactions. <br>NOTE: Don't use it if you don't know.
-Permata `va_number` | String | (optional) | Length should be 10. Only supported for b2b VA type.
-BNI `va_number` | String | (optional)| Length should be within 1 to 8.
-BRI `va_number` | String | (optional)| Length should be within 1 to 13.
+BCA va_number| String | (Optional) | Length should be within 1 to 11.
+BCA sub_company_code | String | (optional) | BCA sub company code directed for this transactions. <br>NOTE: Don't use it if you don't know.
+Permata va_number | String | (optional) | Length should be 10. Only supported for b2b VA type.
+BNI va_number | String | (optional)| Length should be within 1 to 8.
+BRI va_number | String | (optional)| Length should be within 1 to 13.
 
-?> On Production mode, not all Bank support custom VA number, it depends on the agreement, please consult with Midtrans Activation team for further info.
+?> in Production environment, all banks do not  support custom VA number. Please consult Midtrans Activation team for further information.
```

Comment:
This re-wording changes the meaning of the sentence.

> all banks do not  support custom VA number

Implies that every one of the bank does not support it. What we want is to inform: Not every banks may support it. e.g: some support it, some does not.